c++/src/vector.cc

```diff
@@ -11,6 +11,7 @@ class Vector {
   unsigned int length_ = 0;
   unsigned int capacity_ = 8;
   T* dataStore_ = nullptr;
+  const unsigned int growFactor = 2;
```

Comment:
Constants typically have a different naming convention. Usually you go with all caps, underscore-separated: `GROWTH_FACTOR`. Another one is `k`-prefixed camel case: `kGrowthFactor`. The first one is more commonly used; people are more likely to realize that that name represents a constant.